django/chat/models.py

```diff
@@ -138,6 +138,11 @@ def from_defaults(self, mode=None, user=None):
     ("summarize", _("Read entire documents (slow, expensive)")),
 ]
 
+qa_source_order_CHOICES = [
```

Comment:
Conventionally within our codebase, this entire name would be uppercase

Reply:
Whoops, it actually was until a find-and-replace that evidently had some unintended consequences